public/javascripts/songButton.js

```diff
@@ -0,0 +1,146 @@
+var addOrDeleteSong = function(id) {
```

Comment:
In your server side you can do an app.delete as a separate route and make the method 'DELETE'.